code/lab/lab.cpp

```diff
@@ -1424,13 +1425,21 @@ void labviewer_populate_flags_window()
 	// clear out anything that already exists
 	labviewer_flags_clear();
     
+	int y = 0;
+
     // ship flags ...
     if (Lab_mode == LAB_MODE_SHIP) {
-        //Needs reimplementation
+		for (int i = 0; i < Num_ship_flags; ++i) 
```

Comment:
Recommend changing `int i = 0` to `auto i = 0` here since we're getting a signed/unsigned mismatch warning.